### ISSUE TITLE
ULS UNITS VET - Accessibility assessment

### ISSUE THREAD

**Issue by manolobevia** [cul-it/uls#48]
FROM CIT:
One of my designers completed the accessibility assessment for the Vet Library site. He said that overall it was pretty good. However, there was some concern over keyboard and screen reader accessibility.
Additional notes/comment are available here:  
`https://docs.google.com/spreadsheets/d/1j8t5zT_y-46RhTvgQ97HygDyEEzgjnz5LoLlkT3Pf6s/edit?usp=sharing`

**Comment by mhk33** [comment]
I have not viewed the full report yet, but at the media accessibility presentation today they demo'd that when using an accordion/collapse element there should be a visual indicator that the label can be opened, and the visual indicator should also be read by the screenreader, e.g. "click to expand." Just a note to check that this is present on the accordions on the Vet site.

**Comment by mhk33** [comment]
https://fontawesome.com/v4.7.0/accessibility/

**Comment by mhk33** [comment]
I have started adding comments in the "Review Comments" column: https://docs.google.com/spreadsheets/d/1j8t5zT_y-46RhTvgQ97HygDyEEzgjnz5LoLlkT3Pf6s/edit#gid=776733708

**Comment by mhk33** [comment]
Reference for adding aria-expanded attribute to collapsable elements:
https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element

**Comment by mhk33** [comment]
Info about accessible dropdown navigation:
https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

**Comment by manolobevia** [comment]
@mhk33 THANKS for adding all these references.

**Comment by mhk33** [comment]
Holding on any more changes until we implement the new theme.

**Comment by manolobevia** [comment]
Closing this issue since this site will be a child of CULU.
